src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.ProjectSystem.targets

```diff
@@ -22,8 +22,17 @@ Copyright (c) .NET Foundation. All rights reserved.
     <MSBuildWebTargetsPath>$(MSBuildExtensionsPath)\Microsoft\VisualStudio\Managed.Web\</MSBuildWebTargetsPath>
     <AspNetCoreHostingModel Condition="'$(AspNetCoreHostingModel)' == '' And '$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(_TargetFrameworkVersionWithoutV)' != '' And '$(_TargetFrameworkVersionWithoutV)' &gt;= '3.0' ">inprocess</AspNetCoreHostingModel>
     <AspNetCoreModuleName Condition="'$(AspNetCoreModuleName)' == '' And '$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(_TargetFrameworkVersionWithoutV)' != '' And '$(_TargetFrameworkVersionWithoutV)' &gt;= '3.0' ">AspNetCoreModuleV2</AspNetCoreModuleName>
+    <EnableAspNetCoreGlobalUsings Condition="
+      '$(EnableAspNetCoreGlobalUsings)' == ''
+      And '$(TargetFrameworkIdentifier)' == '.NETCoreApp'
+      And $([MSBuild]::VersionGreaterThanOrEquals('$(TargetFrameworkVersion)', '6.0'))
+      And '$(LangVersion)' == 'preview'">true</EnableAspNetCoreGlobalUsings>
```

Comment:
@shirhatti asked me if this will cause a problem for F#, and I'm not sure. Do we add a condition to skip this for fsproj's targeting "Microsoft.NET.Sdk.Web"?

I tried to `<Compile Include=` a DefaultGlobalUsing.cs in an fsproj just to see what happens.

```
FSC : error FS0226: The file extension of 'Microsoft.NET.Sdk.Web.ProjectSystem.DefaultGlobalUsing.cs' is not recognized. Source files must have extension .fs, .fsi, .fsx, .fsscript, .ml or .mli. [C:\dev\halter73\HoudiniPlaygroundFSharp\HoudiniPlaygroundFSharp.fsproj]
```

I did not test with the actual SDK changes, so it's possible I missing some reason this shouldn't affectt fsproj's.

Reply:
The LangVersion check won't pass eventually right, as the C# 10 just becomes the default and is not preview anymore, i.e. when we GA. Also to @halter73's point VB has its own concept of global imports as this is language feature, not general SDK feature, so the condition should ensure it's only applying to C# projects.

Reply:
This is going into the real sdk now anyways.

Reply:
Added the condition for C# only. Still need to check whether we can remove the preview condition though.